pages/restake/lrt-concepts.mdx

```diff
@@ -0,0 +1,53 @@
+# Liquid Restaking on Tangle Network
+
+## Introduction
+
+Liquid restaking allows users to participate in staking while still maintaining liquidity of their staked assets.
+This mechanism is implemented via delegation vaults, enabling participants to stake their tokens in a vault, earn rewards, and retain the ability to unbond or transfer staked assets without waiting for the traditional staking lock-up periods.
```

Comment:
Is this accurate? Seems over complicated. It's implemented through smart contracts that define their own mechanics but proxy all calls through the contract.